### PR TITLE
4.9 Admin docs

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -738,6 +738,22 @@ platform user deleteall
...
+
+  Format
+    .. code-block:: none
+
+       platform user email {user} {email}
+
+  Example
+    .. code-block:: none
+
+      sudo ./platform user email user@example.com user@example.com
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: Is {user} the current email, and {email} the new email (?)

File: source/administration/compliance-export.rst [hunk 2]
```diff
@@ -32,6 +32,8 @@ Actiance Vantage Integration
 
 If you have chosen your file format to be Actiance XML, you can set up an integration with Actiance Vantage.
 
+Channel names will now be exported in the Actiance XML file with the channel type prepended.
```
Comment (jasonblais) [inline source/administration/compliance-export.rst]: This document (compliance-export.rst) needs a few updates:
 - throughout, we say only Actiance is supported, but now both Actiance and Global Relay are supported (see introduction section, and step 3 of the set up guide)
 - there are additional settings for Global Relay to configure, which are not mentioned
 - there is no mention of how to configure it for Global Relay

As an additional note, we may need to revise the flow of the doc: right now it's Introduction > Set Up Guide > Actiance > Troubleshooting, but we'll want to support the flow for Global Relay as well

File: source/administration/compliance.rst [hunk 3]
```diff
@@ -135,10 +135,14 @@ Compliance query results stored in ``posts.csv`` file
 | PostFilenames       | Comma separated list of filesnames attached to post           | ["/f../ho.png","/f../hi.png"] |
 +---------------------+---------------------------------------------------------------+-------------------------------+
 
-Global Relay Support 
-====================
+Global Relay Support (Beta)
```
Comment (jasonblais) [inline source/administration/compliance.rst]: We didn't make any changes to the "Compliance" feature, only to "Compliance Exports".

Propose we add these changes to `compliance-export.rst` (see above note) + link to the new `compliance-export.rst` doc in here.

File: source/install/smtp-email-setup.rst [hunk 4]
```diff
@@ -111,6 +111,13 @@ Hotmail
 -  Set **SMTP Port** to **587**
 -  Set **Connection Security** to **STARTTLS**
 
+Office365 / Outlook
+^^^^^^^^^^^^^^^^^^^^^
+
+- Set **SMTP Server Name** to **smtp.office365.com**
```
Comment (jasonblais) [inline source/install/smtp-email-setup.rst]: 1) Can we make these setting names consistent with the other servers? I'm not sure what "encryption method" is, and how that maps to Mattermost settings. Also, it's missing the username/password fields.
2) Did we confirm we've tested with these settings?

File: source/administration/compliance-export.rst (hunk 2 above)
Comment (amyblais) [reply source/administration/compliance-export.rst]: @jasonblais Made some initial edits, but can you help with ` additional settings for Global Relay to configure` - I can't find them from System Console and I also have hard times filling in config settings doc for Global Relay settings.

Comment (jasonblais) [reply source/administration/compliance-export.rst]: @amyblais Once you've chosen Global Relay in the compliance exports, you'll find the config settings

image

Comment (amyblais) [reply source/administration/compliance-export.rst]: @jasonblais I tried earlier (on pre-release with my admin account and on ci-linux with bot@mattermost account) and the drop-down doesn't work - what account should I use to access?

**Comment by amyblais** [comment]
@jasonblais FYI, this was still WIP.

**Comment by jasonblais** [comment]
@amyblais Ah sorry..I misread you earlier comment, I see you now say you made initial edits.

**Comment by lindy65** [comment]
@jasonblais - I'm not able to add Carlos as a reviewer.  Are you able to add him here on GitHub?

**Comment by amyblais** [comment]
@jasonblais For smtp-email-setup.rst, Outlook/Office365 is tested but there are some issues that will be fixed for 4.10 or we do a dot release - do I still include the settings?

**Comment by jasonblais** [comment]
@amyblais If we cannot confirm it's working, we shouldn't document nor changelog it and instead do that in the next release (or whenever it's fixed).

**Pull request merged**